lib/engine-test.js

```diff
@@ -52,7 +52,7 @@ EngineTest.prototype.getVersion = function(callback) {
 
   var result = '';
   var flag = ENGINE_TEST_ARGS[this.type] || '-v';
-  var version = childProcess.spawn(this.type, [flag]);
+  var version = crossSpawn.spawn(this.type, [flag]);
```

Comment:
In `5.0.0`, [work was done to support the `shell` option](https://github.com/IndigoUnited/node-cross-spawn/pull/46). Should that be passed as an option here?